Core/src/SplineInterpretation/SplineInterpretation.cpp

```diff
@@ -368,10 +368,11 @@ size_t OpenInfraPlatform::Core::SplineInterpretation::SplineInterpretation::vari
 		gamma[h - 1] /= (2 * N);

```

Comment:
```suggestion
		gamma[h - 1] /= (2. * N);
```

It is advisable to use floating numbers when dividing, so that the result is not rounded.

Reply:
Done in commit e4fc298.